### ISSUE TITLE
ModuleNotFoundError: No module named 'moviebot.core.utterance'

### ISSUE THREAD

**Issue by mgskjaeveland** [iai-group/MovieBot#105]
Testing documentation in #87 results in error. 

1.  `pip3 install -r requirements.txt`, finishes successfully
2.  `python3 -m moviebot.run -c config/moviebot_config.yaml` gives the following output

```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/martige/repo/misc/moviebot/moviebot/run.py", line 8, in <module>
    from moviebot.controller.controller_messenger import ControllerMessenger
  File "/home/martige/repo/misc/moviebot/moviebot/controller/controller_messenger.py", line 12, in <module>
    from moviebot.controller.controller import Controller
  File "/home/martige/repo/misc/moviebot/moviebot/controller/controller.py", line 8, in <module>
    from moviebot.core.utterance.utterance import UserUtterance
ModuleNotFoundError: No module named 'moviebot.core.utterance'
```

**Comment by NoB0** [comment]
This should be fixed by PR #79.

**Comment by mgskjaeveland** [comment]
My bad, I tested with branch #87.
